src/main/java/reactor/netty/transport/ProxyProvider.java

```diff
@@ -389,6 +406,15 @@ public ProxyProvider build() {
 		 */
 		Builder httpHeaders(Consumer<HttpHeaders> headers);
 
+		/**
+		 * The proxy connect timeout in millis. Default to 10000 ms.
+		 * If this value set as non positive value, 0 ms will be applied.
```

Comment:
> If this value set as non positive value, 0 ms will be applied.

May be something like this is better `If this value set as non positive value, there is no connect timeout.`